src/main/java/uk/gov/ons/fwmt/job_service/exceptions/types/CSVInvalidFieldException.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.ons.fwmt.job_service.exceptions.types;
+
+import uk.gov.ons.fwmt.job_service.exceptions.ExceptionCode;
+
+public class CSVInvalidFieldException extends FWMTCommonException {
```

Comment:
Do we need this class since we are not adding any behaviour in this class that is different from Parent Class.
Call the parent class directly with ErrorCode and cause.
Add the class in case we are adding new behaviour.

Remove other exception class which does the same.

Reply:
IMO, we don't need any subclass created for Exception if we are not any creating any business functionality based on exception.